### PR TITLE
Add c9x.me

### PR DESCRIPTION
**Pull request by mpu**
Hi, I own the domain and wanted to add it to your ruleset.
Thanks!

### CODE REVIEW

File: src/chrome/content/rules/C9x.me.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+	Nonfunctional hosts in *.c9x.me:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/C9x.me.xml]: These are unnecessary, please delete comments that don't have corresponding subdomains.

File: src/chrome/content/rules/C9x.me.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!--
+	Nonfunctional hosts in *.c9x.me:
+
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/C9x.me.xml]: Invalid certificate:
www.c9x.me
ssl.c9x.me

**Comment by mpu** [comment]
Hey, so I eventually made the changes required.
Let me know if this version still has problems.

**Comment by gloomy-ghost** [comment]
Thanks. You only need to note [problematic subdomains](https://github.com/EFForg/https-everywhere/pull/7516#discussion_r87668113) in the top comment.

Also please:
- [ ] Add `<securecookie host=".+" name=".+" />` between `target`s and `rule`s

**Comment by gloomy-ghost** [comment]
@jeremyn @J0WI I've pushed a commit for this backlog PR, could one of you have a look?

**Comment by J0WI** [comment]
LGTM, Thanks!

**Pull request merged**